bond_test.go

```diff
@@ -51,10 +63,23 @@ type User struct {
 	Username  string `db:"username"`
 }
 
+func (u User) AfterCreate(tx bond.Session) {
```

Comment:
This other `AfterCreate()` expects a session, this session can be a normal session or a transaction, depending on the context *of the caller*. Rows created by this `AfterCreate()` will be discarded in case of a rollback.